CoreWelcomeView/test/unit/src/au/gov/asd/tac/constellation/views/welcome/WelcomeTopComponentNGTest.java

```diff
@@ -64,4 +68,16 @@ public void testCreateStyle() {
 
         assertEquals(instance.createStyle(), expResult);
     }
+    
+    /**
+     * Test of createContent method, of class WelcomeTopComponent.
+     */
+    @Test
+    public void testCreateContent() {
+        System.out.println("createContent");
+        final WelcomeTopComponent instance = spy(WelcomeTopComponent.class);
+        WelcomeViewPane pane = instance.createContent();
+        assertNotNull(pane);
+        assertTrue(pane.getBottomRecentSection() instanceof HBox);
```

Comment:
is there anything else that can be asserted here? I'm not sure if this set of asserts on its own says much about the actual function itself.